src/main/java/com/shoppinglist/shoppinglist/service/ShoppingListService.java

```diff
@@ -2,15 +2,19 @@
 
 import com.shoppinglist.shoppinglist.dao.IShoppingListDAO;
 import com.shoppinglist.shoppinglist.dto.ListItem;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Component;
 
 import java.util.List;
 import java.util.NoSuchElementException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 @Component
 public class ShoppingListService implements IShoppingListService {
     // Todo: Add cache annotations
     private final IShoppingListDAO shoppingListDAO;
+    private static final Logger logger = LoggerFactory.getLogger(ShoppingListService.class);
 

```

Comment:
A logger was created that can be used to log various log messages, such as error messages. It can help to analyze failures later.